src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -70,6 +70,7 @@
     toggle-expand="toggleExpandPanel"
     create-child-ui-state="createChildUiState"
     toggle-expand="toggleExpandPanel"
+    reporting-items-count="{{state.panels.length}}"
```

Comment:
That was the initial approach that I went, but it ends up that the dashboard's use of gridster incrementally adds the elements to the DOM, and based on the timing, it was sometimes only returning some of them.